docs/book/src/user/quick-start.md

```diff
@@ -678,6 +678,23 @@ kubectl --kubeconfig=./capi-quickstart.kubeconfig get nodes
 {{#/tab }}
 {{#/tabs }}
 
+### Clean Up
+
+Delete workload cluster and wait for it to be completely cleaned up.
+```bash
+kubectl delete -f capi-quickstart.yaml
```

Comment:
This should be 
```suggestion
kubectl delete cluster capi-quickstart
```
Because deleting the cluster ensure that the deletion of all the components happens in the right order

Reply:
Thanks. Done.